libs/core/langchain_core/runnables/graph_mermaid.py

```diff
@@ -16,6 +16,7 @@ def draw_mermaid(
     edges: List[Edge],
     first_node_label: Optional[str] = None,
     last_node_label: Optional[str] = None,
+    with_styles: bool = True,
```

Comment:
Same here -- it's a breaking change in the API

I'd love if we used `/` and `*` operators on all function signatures